app/controllers/accounts_controller.rb

```diff
@@ -6,7 +6,7 @@ def edit
 
   def update
     changed_notes = []
-    if @user.name != account_params[:name]
+    if @user.first_name != account_params[:first_name] || @user.last_name != account_params[:last_name]
```

Comment:
Thanks!  These have been only partially solved since the beginning of the project when we had one field.